### PR TITLE
Detail metrics name changes

### PR DESCRIPTION
**Pull request by miekg**
Add more detail in the notes to help our users.

### CODE REVIEW

**Comment by miekg** [comment]
@SuperQ ptal, as I might have missed some. I think we dropped a metric or 2 as well

**Comment by SuperQ** [comment]
`coredns_dns_request_type_count_total` is now part of `coredns_dns_requests_total`.

File: notes/coredns-1.7.0.md [hunk 1]
```diff
@@ -17,5 +17,39 @@ This is a **backwards incompatible release**. Major changes include:
...
+* `coredns_forward_request_count_total` -\> `coredns_forward_requests_total`
+* `coredns_forward_response_rcode_count_total` -\> `coredns_forward_responses_total`
+* `coredns_forward_healthcheck_failure_count_total` -\> `coredns_forward_healthcheck_failures_total`
+* `coredns_forward_healthcheck_broken_count_total` -\> `coredns_forward_healthcheck_broken_total`
+* `coredns_forward_max_concurrent_reject_count_total` -\> `coredns_forward_max_concurrent_rejects_total`
+
+* `coredns_grpc_request_count_total` -\> `coredns_grpc_requests_total`
+* `coredns_grpc_response_rcode_count_total` -\> `coredns_grpc_responses_total`
+
+* `coredns_panic_count_total` -\> `coredns_panics_total{}`
```
Comment (SuperQ) [inline notes/coredns-1.7.0.md]: Drop the {} here.

```suggestion
* `coredns_panic_count_total` -\> `coredns_panics_total`
```

**Comment by miekg** [comment]
this needs more work before releasing anyway, merging.

**Pull request merged**